tokio/CHANGELOG.md

```diff
@@ -1,3 +1,62 @@
+# 1.30.0 (August 6, 2023)
+
+tokio: bump MSRV to 1.63 ([#5887])
+
+### Fixed
+- runtime: fix flaky test `wake_while_rt_is_dropping` ([#5905])
+
```

Comment:
```suggestion
```

As said in https://github.com/tokio-rs/tokio/pull/5917#discussion_r1285368188, this is a change in the test and does not affect users.

Reply:
Sorry, missed that